frontend/src/components/Layout/index.tsx

```diff
@@ -8,15 +8,42 @@ import TopNav from "./TopNav";
 import { SelectedClassroomContext } from "@/contexts/selectedClassroom";
 
 import "./styles.css";
+import { FaTachometerAlt } from "react-icons/fa";
+import { MdEditDocument } from "react-icons/md";
+import { MdFactCheck } from "react-icons/md";
+import { FaGear } from "react-icons/fa6";
+import { useClassroomUser } from "@/hooks/useClassroomUser";
+import { ClassroomRole } from "@/types/users";
 
 const Layout: React.FC = () => {
   const { selectedClassroom: selectedClassroom } = useContext(
     SelectedClassroomContext
   );
+  const { classroomUser } = useClassroomUser(selectedClassroom?.id);
+
+  const baseNavItems = [
+    { name: "Dashboard", dest: "/app/dashboard", Icon: FaTachometerAlt },
+    { name: "Grading", dest: "/app/grading", Icon: MdEditDocument },
+  ];
+
+  const professorNavItems = [
```

Comment:
Is there an instance where a TA might need to View a rubric? (Not Edit access only View)?

Reply:
I think we've concluded that TAs should only see rubrics in the context of grading an assignment